ruby/vampires.rb

```diff
@@ -23,18 +23,23 @@
 
 
 
-	if allergies =! "done"
+	until allergies == "done" or allergies == "sunshine"
 
 		puts "Do you have any suspicious allergies?"
 		allergies = gets.chomp
 
 		if allergies == "sunshine"
 			puts "Probably a vampire"
+			allergies = "done"
 		end
 
 		
 	end
 
+
+
+	p "Actually, never mind! What do these questions have to do with anything? Let's all be friends."
+
 			
 
 
```

Comment:
When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`